resource/types.go

```diff
@@ -20,15 +20,28 @@ const (
 	notinStr = "notin"
 )
 
+var _ PatchSetMetadata = (*DataChangeEvent)(nil)
+
 // DataChangeEvent represents a record of a change made to a database table.
 type DataChangeEvent struct {
 	TableName   accesstypes.Resource `spanner:"TableName"`
 	RowID       string               `spanner:"RowId"`
+	Sequence    int                  `spanner:"Sequence"`
 	EventTime   time.Time            `spanner:"EventTime"`
 	EventSource string               `spanner:"EventSource"`
 	ChangeSet   spanner.NullJSON     `spanner:"ChangeSet"`
 }
 
+// Resource returns the Resource name for DataChangeEvent
+func (d DataChangeEvent) Resource() accesstypes.Resource {
```

Comment:
Why not use a nameless receiver instead of adding an exception to the linter?

```suggestion
func (DataChangeEvent) Resource() accesstypes.Resource {
```